tests/fixtures/course.json

```diff
@@ -1391,5 +1391,65 @@
 			}
 		],
 		"status_code": 200
+	},
+	"add_grading_standards": {
+		"method": "POST",
+		"endpoint": "courses/1/grading_standards",
+		"data": {
+			"title": "Grading Standard 1",
+			"id": 1,
+			"context_id": 1,
+			"context_type": "Course",
+			"grading_scheme": [
+			  {"name": "A", "value": 0.9},
+			  {"name": "B", "value": 0.8},
+			  {"name": "C", "value": 0.7}
+			]
+		  },
+		"status_code": 200
+	},
+	"get_grading_standards": {
+		"method": "GET",
+		"endpoint": "courses/1/grading_standards",
+		"data": [
+			{
+			"title": "Grading Standard 1",
+			"id": 1,
+			"context_id": 1,
+			"context_type": "Course",
+			"grading_scheme": [
+			  {"name": "A", "value": 0.9},
+			  {"name": "B", "value": 0.8},
+			  {"name": "C", "value": 0.7}
+			]
+		  },
+		  {
+			"title": "Grading Standard 2",
+			"id": 2,
+			"context_id": 2,
```

Comment:
This `context_id` combines with `context_type` to reference the parent object, in this case Course #1. Update `context_id` to reflect this.